### PR TITLE
made processing version of CC 16

### PR DESCRIPTION
**Pull request by TheLastDestroyer**
for https://github.com/CodingTrain/website/issues/267

capture
capture1

### CODE REVIEW

File: CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+String sentence = axiom;
+int len = 100;
+Rule[] rules;
+
+
+void setup(){
+  rules = new Rule[1];
+  rules[0] = new Rule('F', "FF+[+F-F-F]-[-F+F+F]");
+  
+  size(400,400);
```
Comment (shiffman) [inline CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde]: `size()` should always be first in `setup()` in Processing.

File: CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde [hunk 2]
```diff
@@ -0,0 +1,82 @@
+float angle;
```
Comment (shiffman) [inline CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde]: Can you add header comments?

**Comment by TheLastDestroyer** [comment]
that should be the changes made

File: CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde (hunk 2 above)
Comment (TheLastDestroyer) [reply CodingChallenges/CC_016_LSystem_Processing/CC_016_LSystem_Processing.pde]: added

**Comment by shiffman** [comment]
Wonderful!

**Pull request merged**